src/Core/DynamicMetadata/CropArea.php

```diff
@@ -22,7 +22,7 @@ public static function getName()
      */
     public static function createFromJsonResponse($data, $isArray = false)
```

Comment:
what is the idea behind the `$isArray` flag when that is clear from the provided data itself?
I would also recommend to use different methods for the already decoded data and the json data, e.g.
`createFromJsonResponse(string $data)` and `createFromDecodedJsonResponse(array $data)`

Reply:
I actually wondered that myself why there is this "$isArray" parameter, @ChristianRiesen maybe you know? 
But you're right, we should create two functions for that, even though it's kinda of a BC break, but I don't think anyone is using them, since they're there for internal usage

Reply:
This goes back all the way to Matteo's days :) Making that two functions seems more sensible to me as well. Maybe check with the wordpress and the drupal plugins to see if either of them uses this.

Reply:
Did now the separation of createFromDecodedJsonResponse and createFromJsonResponse. Also checked the other plugins/extensions of this class, they don't use those methods.